src/main/java/se/sundsvall/casedata/service/util/mappers/EntityMapper.java

```diff
@@ -521,9 +522,13 @@ public static Notification toNotification(final NotificationEntity notificationE
 	}
 
 	public static String toOwnerId(final ErrandEntity errandEntity) {
+		return getUserIdFromRole(errandEntity, ADMINISTRATOR);
+	}
+
+	public static String getUserIdFromRole(final ErrandEntity errandEntity, final StakeholderRole stakeholderRole) {
 		return ofNullable(errandEntity.getStakeholders()).orElse(emptyList()).stream()
 			.filter(stakeholder -> ofNullable(stakeholder.getRoles()).orElse(emptyList()).stream()
-				.anyMatch(ADMINISTRATOR.toString()::equalsIgnoreCase))
+				.anyMatch(stakeholderRole.toString()::equalsIgnoreCase))
```

Comment:
You are mixing toString() and name() when extracting the enum value string representations. Is this by intention?

(In ErrandReporterNotificationProcessor.java‎ you are using name())

Reply:
This is "old" code (which is refactored as I needed to fetch stakeholders with other roles as well) and I didn't notice that it used toString. I'll change so that it also uses name.